molior-deploy

```diff
@@ -525,57 +540,59 @@ setup_deployment()
   BASEVERSION=`basename $MIRROR`
   log " * mirror: $MIRROR"
 
+  log_info "Getting source package ..."
   if [ -n "$SOURCE_DIR" ]; then
     SOURCE_DIR=`readlink -f $SOURCE_DIR`
     log " * using local package source: $SOURCE_DIR"
     cd $SOURCE_DIR
     REVISION=`dpkg-parsechangelog -S Version`+local
-    log " * found revision: $PACKAGE_NAME $REVISION"
+    log " * found source package: $PACKAGE_NAME $REVISION"
     cd - >/dev/null
   else
-    log_info "Searching latest $PACKAGE_NAME revision ..."
     SOURCES_VERSION=$VERSION
     if [ -n "$OVERLAY_VERSION" ]; then
       SOURCES_VERSION=$OVERLAY_VERSION
-      log " * Using overlay source: $SOURCES_VERSION"
+      log " * using overlay source: $SOURCES_VERSION"
     fi
     if [ "$OVERLAY_AUTO" -eq 1 ]; then
       baseurls=`echo "$APT_SOURCES" | grep $MIRROR/repos/$PROJECT | awk '{print $2}'`
       for baseurl in $baseurls
       do
         SOURCES_VERSION=`basename $baseurl`
-        log " * Trying to download package source from $SOURCES_VERSION"
+        log " * trying to download package source from $SOURCES_VERSION"
         SOURCES=`wget -q -O- $MIRROR/repos/$PROJECT/$SOURCES_VERSION/dists/$DIST/main/source/Sources`
         if [ -z "$SOURCES" ]; then
+          # try next baseurl
           continue
         fi
-        FILENAME=`echo "$SOURCES" | /usr/lib/molior-tools/find-latest-version.pl $PACKAGE_NAME $PACKAGE_VERSION`
+        t=`echo "$SOURCES" | /usr/lib/molior-tools/find-latest-version.pl $PACKAGE_NAME $PACKAGE_VERSION`
+        FILENAME=`echo $t | cut -d ' ' -f 1`/`echo $t | cut -d ' ' -f 2`
+        REVISION=`echo $t | cut -d ' ' -f 3`
         if [ -n "$FILENAME" ]; then
+          # found package
           break
         fi
       done
       if [ -z "$FILENAME" ]; then
         log_error "No filename found for $PACKAGE_NAME in $MIRROR/repos/$PROJECT/$SOURCES_VERSION/dists/$DIST/main/source/Sources"
       fi
-      log " * Using package source: $SOURCES_VERSION"
+      log " * using package source: $SOURCES_VERSION"
     else
       SOURCES=`wget -q -O- $MIRROR/repos/$PROJECT/$SOURCES_VERSION/dists/$DIST/main/source/Sources`
       if [ -z "$SOURCES" ]; then
         log_error "Could not download $MIRROR/repos/$PROJECT/$SOURCES_VERSION/dists/$DIST/main/source/Sources"
       fi
-      FILENAME=`echo "$SOURCES" | /usr/lib/molior-tools/find-latest-version.pl $PACKAGE_NAME $PACKAGE_VERSION`
+      t=`echo "$SOURCES" | /usr/lib/molior-tools/find-latest-version.pl $PACKAGE_NAME $PACKAGE_VERSION`
```

Comment:
Could you give it a better name than `t`

Reply:
done